dev/widgets.py

```diff
@@ -45,10 +49,17 @@ def __init__(self, parent, text, width=None):
     def update_text(self, text):
         self.set_text(text)
 
+    def _set_font_size(self, font_size):
+        style = lv.style_t()       
+        lv.style_copy(style, self.get_style(lv.label.STYLE.MAIN))
+        if font_size == 28:
```

Comment:
What is going on here?

Reply:
In order to change the font of the text in the label you need to design a new style with the required font size. I copy the font style currently in use, and then update for the only larger size available with LittlevGL (roboto 16 and 28 are the only options available with micropython). We can think about adding our own fonts and sizes further down the line.

Reply:
Okay, cool